src/scss/patterns/_icon-blocks.scss

```diff
@@ -1,9 +1,21 @@
-.icon-blocks {
-  --media-size: calc(var(--extra) - var(--gutter));
+[icon-grid] {
   --fill: var(--icon);
+}
+
+[icon-grid='full'] {
+  --media-size: calc(var(--extra) - var(--gutter));
   --column: full;
 }
 
+[icon-grid='fit'] {
```

Comment:
I wonder if we want to allow some icon blocks, like this one, to have left aligned (but still on top) icons as it looks a bit off balanced if we have smaller amounts of text at small (not tiny) screens 
![Screenshot 2024-04-03 at 2 16 54 PM](https://github.com/oddbird/oddleventy/assets/1581694/7fd6d32f-89d0-444d-ac46-132a1bdccd45)